routes/books.js

```diff
@@ -1,8 +1,63 @@
 const express = require("express");
 const generateId = require("uuid/v4");
+const books = require("../data/books");
+const authors = require("../data/authors");
+const body = require('body-parser')
 
 const router = express.Router();
 
 // Define your endpoints here
 
-module.exports = router;
+
+router.get('/', (req, res, next) => {
+	res.json(books)
+})
+
+router.get('/:id', (req, res, next) => {
+	res.json(books.filter(el => el.id === req.params.id))
+})
+
+router.post('/', (req, res, next) => {
+	const { author, title, description } = req.body
+	const book = {
+		id: generateId,
+		author,
+		title,
+		description
+	}
+	books.push(book)
+	res.json(book)
+})
+
+router.delete('/:id', (req, res, next) => {
+	books.splice(books.indexOf(books.filter(el => el.id === req.params.id)))
+	res.status(200)
```

Comment:
Nice thinking. What you want though is `res.sendStatus(200)`. An even better HTTP code would be 202.

Reply:
Thanks for the tip !